packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -796,11 +881,29 @@ class Scaffold extends StatefulWidget {
     this.resizeToAvoidBottomPadding,
     this.resizeToAvoidBottomInset,
     this.primary = true,
+    this.extendBody = false,
     this.drawerDragStartBehavior = DragStartBehavior.down,
   }) : assert(primary != null),
+       assert(extendBody != null),
        assert(drawerDragStartBehavior != null),
        super(key: key);
 
+  /// If true and [bottomNavigationBar] or [persistentFooterButtons]
+  /// are specified then the [body] extends to the bottom of the Scaffold,
+  /// instead of only extending to the top of the [bottomNavigationBar]
+  /// or the [persistentFooterButtons].
+  ///
+  /// If true a [MediaQuery] widget whose bottom padding matches the
```

Comment:
```suggestion
  /// If true, a [MediaQuery] widget whose bottom padding matches the
```
(at least I think so)

Reply:
Sounds good to me.